en/flight_modes/return.md

```diff
@@ -15,26 +15,52 @@ The specific behaviour for each vehicle type is described below.
 
 ## Multi-Copter (MC)
 
-A copter/multi-rotor will first ascend to the `RTL_RETURN_ALT` altitude and then fly to the home position in a straight line (if already above `RTL_RETURN_ALT` it will return at its current altitude).
+A copter/multi-rotor will first ascend to the return altitude (*by default*, [RTL_RETURN_ALT](#RTL_RETURN_ALT)) and then fly to the home position in a straight line and constant altitude (if already above the return altitude it will return at its current altitude).
 
-When it arrives at the home/launch position it will rapidly descend to the [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) altitude. It will then hover for the period defined in [RTL_LAND_DELAY](#RTL_LAND_DELAY) before landing.
+When it arrives at the home/launch position it will rapidly descend to the [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) altitude.
+It will then hover for the period defined in [RTL_LAND_DELAY](#RTL_LAND_DELAY) before landing.
+
+The required return altitude can be configured using using the parameter [RTL_CONE_ANG](#RTL_CONE_ANG), which defines the angle of a half cone centered around the home position.
+
+![Return mode cone](../../assets/flying/rtl_cone.jpg)
+
+<!-- Original draw.io diagram can be found here: https://drive.google.com/file/d/1W72XeZYSOkRlBSbPXCCiam9NMAyAWSg-/view?usp=sharing -->
+
+If the vehicles is:
+- Above [RTL_RETURN_ALT](#RTL_RETURN_ALT) (1) it will return at its current altitude.
+- Below the cone it will return where it intersects the cone (2) or [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) (whichever is higher).
+- Outside the cone (3) it will first climb until it reaches [RTL_RETURN_ALT](#RTL_RETURN_ALT).
+- Inside the cone:
+  - Above [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) (4) it will return at its current altitude.
+  - Below [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) (5) it will first ascend to `RTL_DESCEND_ALT`.
+
+Note:
+- If [RTL_CONE_ANG](#RTL_CONE_ANG) is 0 degrees there is no "cone":
+  - the vehicle returns at `RTL_RETURN_ALT` (or above).
+  - This is the default behaviour.
+  - This is the only behaviour prior to PX4 v1.9.

```

Comment:
@hamishwillee I don't think we have this feature in 1.9, do we?

Reply:
@RomanBapst What I was trying to say is that there is no cone in PX4 v1.9 - you just return at RTL_RETURN_ALT or higher.

Reply:
I have removed this. They can read the old docs to find out the old behaviour.